.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# rules are explained here
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @zjswhhh will be requested for review when someone opens
+# a pull request.
+*       @zjswhhh
```

Comment:
Please add Longxiang and me. Thanks.

Reply:
Hi @yxieca, added Longxiang. I hope you understand that

1. Repo ownership comes with responsibility of PR review and repo maintenance. 
2. Owners should have enough code contributions to fulfill that responsibility.